src/pages/ProductListing/ProductListing.jsx

```diff
@@ -1,10 +1,21 @@
+import "./ProductListing.css";
 import { useDocumentTitle } from "../../customHooks/useDocumentTitle";
-
+import { Card } from "../../components";
+import { products } from "../../backend/db/products";
+import { Filters } from "../../pages";
 export const ProductListing = () => {
 	useDocumentTitle("Products | MS");
 	return (
-		<div className="body--homepage">
-			<h2>ProductListing Page</h2>
+		<div className="product_page_wrapper">
+			<Filters />
+			<main className="main main--products">
+				<section className="products__section items_container">
+					<h2 className="h3 section__heading">Showing All Products</h2>
+					<ul className="categories__items">
+						<Card products={products} />
```

Comment:
You can  directly map the Card here instead of doing it inside of Card component.

what i mean by it.


`data.map(item=> <Card props={item} />)`

Reply:
Actually, I am reusing the card component to map products on the home page aswell.